### PR TITLE
Expand GCAM overview

### PR DESCRIPTION
**Pull request by rplzzz**
This is an update to the model overview section of the docs that was written by an intern a little over a year ago.  I'm not sure where he got the information.  It doesn't read like an intern's writing, so I'd conjecture that someone unfamiliar with the documentation system wrote it and asked him to upload it.  Either way, it looks pretty good.  Please take a look and let me know if you think it would be a good addition when we update for the v4.4 release.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
...
-mitigation policy.
-
-## Documentation
+## GCAM Model Overview
+
+### Introduction
+
+GCAM is a global integrated assessment model that represents the
+behavior of, and complex interactions between five systems: the energy
+system, water, agriculture and land use, the economy, and the
```
Comment (kvcalvin) [inline index.md]: Should we remove "water" from this line since it isn't in GCAM4.3?

File: index.md [hunk 2]
```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
...
+climate. GCAM has been under development for over 30 years. Work began
+in 1980 with the work first documented in 1982 in working papers
+(Edmonds and Reilly, 1982a,b,c) and the first peer-reviewed
+publications in 1983 (Edmonds and Reilly, 1983a,b,c). A this point,
+the model was known as the Edmonds-Reilly (and subsequently the
+Edmonds-Reilly-Barnes) model. The model was renamed MiniCAM in the
+mid-1990s, and the model was renamed to GCAM in the mid-2000s. The
```
Comment (kvcalvin) [inline index.md]: Renaming to GCAM was post-RCPs, so probably around 2010. Not sure if we need to be that specific.

File: index.md [hunk 3]
```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
...
+first coupling to a carbon cycle model was published in Edmonds, et
+al. (1984). The first use of GCAM (MiniCAM at the time) in conjunction
+with a Monte Carlo uncertainty analysis was published in Reilly,
+Edmonds, Gardner and Brenkert (1987).
+
+Throughout its lifetime, GCAM has evolved in response to the need to
+address an expanding set of science and assessment questions. The
+original question that the model was developed to address was the
+magnitude of mid-21st-century global emissions of fossil fuel
+CO2. Over time GCAM has expanded its scope to include a wider set of
```
Comment (kvcalvin) [inline index.md]: Can we make subscripts?

File: index.md [hunk 4]
```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
...
+energy producing, transforming, and using technologies, emissions of
+non-CO2 greenhouse gases, agriculture and land use, water supplies and
```
Comment (kvcalvin) [inline index.md]: Subscript?

Comment (kvcalvin) [reply index.md]: Also, water?

File: index.md [hunk 5]
```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
...
+demands, and physical Earth systems. It is increasingly being used in
+multi-model, multi-scale analysis, in which it is either soft- or
+hard-coupled to other models with different foci and often greater
+resolution in key sectors. For example, it has been coupled to a state
+of the art Earth system model (Collins, et al., 2015). GCAM has been
```
Comment (kvcalvin) [inline index.md]: Could add link to iESM repo here if we wanted (github.com/ACME-Climate/iESM)

File: index.md [hunk 6]
```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
...
+used to produce scenarios for national and international assessments
+ranging from the very first IPCC scenarios (Response Strategies
+Working Group, 1990) through the present Shared Socioeconomic Pathways
+(Ebi, et al., 2014). Hundreds of papers have been published in
```
Comment (kvcalvin) [inline index.md]: Maybe better to update to the SSP IAM papers (Calvin et al., 2017)

Comment (kvcalvin) [reply index.md]: Also, do we need to actually add these references?

File: index.md [hunk 7]
```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
...
+GCAM is an integrated assessment model. The role of integrated
+assessment research and integrated assessment modeling is to bring
+multiple human and physical Earth systems together in one place to
+shed light on system interactions and provide scientific insights that
+would not otherwise be available from the pursuit of traditional
+disciplinary scientific research alone. IA models are constructed to
+explore these interactions in a single computational platform with a
+sufficiently low computational requirement to allow for broad
+explorations of scenarios and uncertainties. IA models include both
+human and physical Earth systems, Figure 1. Components of an IA model
```
Comment (kvcalvin) [inline index.md]: Where is figure 1?

File: index.md (hunk 1 above)
Comment (rplzzz) [reply index.md]: This won't be merged until the GCAM 4.4 release, if that makes any difference.

File: index.md [hunk 8]
```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
...
+understanding the physical and economic details of human and physical
+Earth system interactions. GCAM is a member of the latter class of
+more highly-resolved IA models.
+
+IA models like GCAM, as with any model, are not predictors of the
+future. They are used to provide conditional forecasts of the
+future. GCAM takes in external “scenario assumptions” about key
+drivers such as population, economic activity, technology, and
+policies, analyzes the implications of these through representations
+of human and physical systems operate and interact, and then developed
```
Comment (kvcalvin) [inline index.md]: This sentence needs editing.

**Review by kvcalvin — DISMISSED** [review]
Small comments throughout

File: index.md (hunk 3 above)
Comment (rplzzz) [reply index.md]: Yes.  For future reference, you can include html in markdown, so `CO<sub>2</sub>` renders as CO<sub>2</sub>.

File: index.md (hunk 1 above)
Comment (kvcalvin) [reply index.md]: Don't think so. Water will be in GCAM5 (I think)

File: index.md (hunk 5 above)
Comment (rplzzz) [reply index.md]: That sounds good.

**Comment by rplzzz** [comment]
Thanks @kvcalvin.  I'll revisit these.  I take it you think that the expanded discussion is generally a good idea?  Also, on the subject of water, we are including that in GCAM 4.4, right?  If so, then we might want to leave that in the treatment.

**Comment by kvcalvin** [comment]
GCAM4.4 will not have water.

**Comment by ssmithClimate** [comment]
I have some ideas for wording changes given that our current focus with GCAM. I'll drop by to discuss.

**Comment by rplzzz** [comment]
@ssmithClimate Some of us (including me) are remote today.  Feel free to discuss ideas for changes on the `#gcam-documentation` Slack channel.  I'll be back in the office tomorrow.

File: index.md (hunk 6 above)
Comment (ssmithClimate) [reply index.md]: I'll add the GCAM SSP references and clean them up in general. (I do like having the references, provides a bit of backup for the long history of GCAM.)

**Comment by rplzzz** [comment]
New edits look good to me.  Any other suggestions from anyone?

**Comment by rplzzz** [comment]
Seeing no further suggestions, I'm going to merge this branch into the v4.4 branch.  If anyone thinks of anything else we might like to add, we can always open a new branch off of v4.4.

**Pull request merged**